_people/Sabrina_Shams.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Sabrina Shams
+position: underGradStudent
+avatar: Sabrina_Shams.jpg
+twitter:
+joined: 2020
+---
+
+<img width="300" src="{{site.baseurl}}/images/people/{{page.avatar}}" data-action="zoom">
+
+### Current Project
+My current project is creating informative slide decks for multiple studies and assisting with data cleanup for the BEACH study.
```

Comment:
You could move this sentence about your roles in the lab to the About section I mentioned earlier and just keep a bullet point that says "BEACH study".